adafruit_sht31d.py

```diff
@@ -244,7 +251,7 @@ def _data(self):
             return temperature[0], humidity[0]
         return temperature, humidity
 
-    def _read(self):
+    def _read(self) -> Tuple[float, float]:
```

Comment:
Based on the return of `_data()` this should also be return ofa `Union` of `Tuple[float, float]` and `Tuple[List[float], List[float]]`.